### DISCUSSION TITLE
[FCOS 4.15] Official End of life support

### DISCUSSION THREAD

**Discussion by parseltongued** [okd-project/okd#2054] [Q&A] [answered]
Hi,

Would like to know the official end of life support for FCOS line of OKD, specifically using **4.15.0-0.okd-2024-03-10-010116.**

Thank you.

**Comment by bdlink** [comment, marked as answer]
At the moment, OKD is pursuing a version based on SCOS (CentOS Stream Core OS), anticipating a stable release this month (November 2024).  It is possible, but by no means certain,  that sometime after this is stable, an FCOS version may be revisited.

No versions have official support, just community support (strongly assisted by Red Hat developers).  It is a misunderstanding to ask about the official end of life support for FCOS line of OKD, as there was no official support in the first place.

For new installations, the community suggests using the stable SCOS releases.  For existing FCOS installations, it is recommended to upgrade to the latest FCOS release mentioned in the OP and follow the to-be-finalized guidelines to get onto the SCOS branch and up to the stable SCOS version.
